lochness/xnat/__init__.py

```diff
@@ -10,36 +10,41 @@
 import lochness.tree as tree
 import lochness.config as config
 
-yaml.SafeDumper.add_representer(col.OrderedDict, yaml.representer.SafeRepresenter.represent_dict)
+yaml.SafeDumper.add_representer(
+        col.OrderedDict, yaml.representer.SafeRepresenter.represent_dict)
 
 logger = logging.getLogger(__name__)
 
 @net.retry(max_attempts=5)
 def sync(Lochness, subject, dry=False):
     logger.debug('exploring {0}/{1}'.format(subject.study, subject.id))
 
-    for alias,xnat_uids in iter(subject.xnat.items()):
+    for alias, xnat_uids in iter(subject.xnat.items()):
         Keyring = Lochness['keyring'][alias]
         auth = yaxil.XnatAuth(url=Keyring['URL'], username=Keyring['USERNAME'],
                               password=Keyring['PASSWORD'])
-        for xnat_uid in xnat_uids:
+
+        # CAPS agnostic XNAT data pull
+        for xnat_uid in xnat_uids + [(x[0], x[1].lower()) for x in xnat_uids]:
```

Comment:
xnat_uids = [(project_name_1, subject_id_1), (project_name_2, subject_id_2), (..., ...)]

Reply:
> [('SF','SF1111'), ('CA','CA1111'), ('LA','LA1201')]

Is this a practical example of `xnat_uids`?

Reply:
If yes, you are essentially duplicating the list with lower cases:

> [('SF','SF1111'), ('CA','CA1111'), ('LA','LA1201'), ('SF','sf1111'), ('CA','ca1111'), ('LA','la201')]

Reply:
1. Where is it breaking?
2. You are also looping for upper cases. When you make a get call or XNAT download call, what happens for the upper cases?

Reply:
1. Sorry it's not actively breaking, but makes the loop to do  nothing. Look under `experiments` function
2. Yes, I'm looping both lower and upper cases. If the data for the ID does not exist, `experiments` function returns nothing, resulting in the loop doing nothing.